data_summaries/Bactrian-X.json

```diff
@@ -0,0 +1,320 @@
+{
+    "bactrian-x-en": {
+        "Unique Dataset Identifier": "bactrian-x-en",
+        "Dataset Name": "Bactrian-X (English)",
+        "Paper Title": "Bactrian-X: Multilingual Replicable Instruction-Following Models with Low-Rank Adaptation",
+        "Dataset URL": "https://huggingface.co/datasets/MBZUAI/Bactrian-X",
+        "GitHub URL": "https://github.com/mbzuai-nlp/bactrian-x",
+        "Hugging Face URL": "https://huggingface.co/datasets/MBZUAI/Bactrian-X",
+        "ArXiv URL": "https://arxiv.org/abs/2305.15011",
+        "Semantic Scholar Corpus ID": 258865265,
+        "Collection": "Bactrian-X",
+        "Languages": [
+            "English"
+        ],
+        "Task Categories": [
+            "Question Answering",
+            "Summarization",
+            "Open-Domain Relation Extraction",
+            "Text Classification"
+        ],
+        "Text Sources": [
+            "alpaca",
+            "dolly",
+            "translate.google.com"
+        ],
+        "Model Generated": [
+            "BX LLaMa",
+            "BX BLOOM"
```

Comment:
Not these, as this column is meant to ask what models were used to generate the data